Google.Api.Gax.Grpc/Rest/ReadHttpResponseMessage.cs

```diff
@@ -109,6 +109,14 @@ internal static Rpc.Status CreateRpcStatus(HttpStatusCode statusCode, string con
             }
             try
             {
+                // Error response messages for streaming APIs are embedded within arrays.
+                // If we detect that the content looks like a JSON array, extract it and parse
+                // expecting a single object.
```

Comment:
If there's more than one error, it will fail to parse later on and we'll just end up with the content as the detail.
I don't *think* we can do better than this.